### ISSUE TITLE
Release 4.4.0 - Alpha 1 - DockerListener Integration testing

### ISSUE THREAD

**Issue by CarlosRS9** [wazuh/wazuh#15544]
# Docker Listener module testing

## Modules tests information
|||
|----------------------------------|------ |
| **Main release candidate issue** | #15525     |
| **Version**                      | 4.4.0 |
| **Release candidate #**          | Alpha 1   |
| **Tag**                          | [v4.4.0-alpha1](https://github.com/wazuh/wazuh/tree/v4.4.0-alpha1) |
| **Previous modules tests issue** | -    |

## Test report procedure

All test results must have one the following statuses: 
|                                  |                                                        |
|-----------------|-------------------------------------------------------------------------|
| :green_circle:  | The test passed successfully.                                           |
| :yellow_circle: | The test passed but some enhancements need to be applied to the module. |
| :red_circle:    | The test failed.                                                        |
| :white_circle:  | The test does not apply to the given bucket type or service.            |

Any failing test must be properly addressed with a new issue, detailing the error and the possible cause. It must be included in the Release 4.4.0 project for triage.

The data available in our Wazuh-dev environment for each of these supported services will be used. Additional logs may be created if they are necessary or there is insufficient data for testing.

## Test results

|Test          | Status                  | Issue ref. |
|--------------------------------|-------------------------|------------|
| Test docker listener using ossec.conf | :green_circle:  | -          |

## Auditors validation
The definition of done for this one is the validation of the conclusions and the test results from all auditors.

All checks from below must be accepted in order to close this issue.

- [x] @davidjiglesias
- [x] @CarlosRS9
- [x] @fdalmaup

**Comment by CarlosRS9** [comment]
# Tests results


The [logall_json option](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#logall_json) must be enabled in the `ossec.conf` file before starting the tests.

## Test docker listener using ossec.conf in a manager

Apply the following configuration, restart Wazuh and check ossec.log to verify the module is running
```
<wodle name="docker-listener">
    <interval>30s</interval>
    <attempts>5</attempts>
    <run_on_start>yes</run_on_start>
    <disabled>no</disabled>
</wodle>
```

<details><summary>ossec.log output</summary>

</details>


Run a container with no cache available
```
docker run ubuntu
```

<details><summary>Archives.json output</summary>


</details>


Remove the image

<details><summary>Archives.json output</summary>


</details>


### Conclusions

Pending
